draft-ietf-cbor-7049bis.md

```diff
@@ -936,7 +936,7 @@ the enclosed value to be an integer (or a floating-point value) only.
 ### Bignums {#bignums}
 
 Protocols using tag numbers 2 and 3 extend the generic data model
-({{cbor-data-models}}) with "bignums" representing arbitrarily sized
```

Comment:
I probably would keep the quotes here.  While the term has been introduced before, that was for a different purpose.

Reply:
Agree